### ISSUE TITLE
Strange distortions in PCM tracks.

### ISSUE THREAD

**Issue by kittertron** [ipatix/agbplay#65]
This is something I heard very rarely when using agbplay, but in a couple of songs in some games there would be this weird "warble" like distortion to some PCM tracks. One example is the VS mode theme (track 0000) in Minna no Soft Tetris Advance (right at the 1:08 mark). I also remember hearing it a couple of times in WarioWare Twisted too, I specifically remember the theme for the microgame "Dear John Letter" (track 0712) having heavy distortion, as well as Stumblebot (track 0788), yet a little more minor in the lead instrument . This issue was very rare but seemed important enough to be reported.

**Comment by ipatix** [comment]
I know my response time is a bit slow, but would it be possible to provide a recording of what it sounds like and what it should sound like (i.e. in game recording)?

**Comment by kittertron** [comment]
Here's an example of the distortion; https://youtu.be/ERNP1R6TQUA?si=cDjQO2h1Ag1pNMXn

**Comment by ipatix** [comment]
A couple of things to note: I cannot reproduce the problem in Minna no Soft Tetris. Perhaps the issue has been fixed already?

In regards to Wario Ware Twisted: I currently cannot confirm the issue, since Wario Ware Twisted currently doesn't properly load in agbplay. As I've found out it also uses the same modified sound driver as Metroid Fusion Zero Mission. I first have to fix that before I can do more investigation.

**Comment by ipatix** [comment]
Yeah, once again something that the Metroid sound driver (which is also used in Wario Ware Twisted) does differently. I can't give an estimation when I'll look into this, but it require knowledge of the sound river, which I currently do not have.
